brainstormforce/ultimate-addons-for-gutenberg
# Fix: Styling not applying when used as a reusable block for Advanced Columns, Section & Marketing Button Blocks.

### Description

- trello - https://trello.com/c/pAHu2EUV/199-section-and-marketting-button-color-gradient-not-applying-when-used-as-a-reusable-block

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Create a Section/Marketing button with color/gradient and save as a reusable in master.
- Update and refresh the page you will see the color style is not applying.
- Switch to `reusable-block-issue` branch refresh the editor and issue resolved.

### Checklist:
